automatic/vim/tools/chocolateyuninstall.ps1

```diff
@@ -3,9 +3,10 @@ $installDir = Get-Content "$toolsDir\installDir"
 $shortversion = '90'
 $statement = '-nsis'
 $exeToRun  = "$installDir\vim\vim$shortversion\uninstall.exe"
+
 # From vim-tux.install.  Make input.
 Set-Content -Path "$env:TEMP\vimuninstallinput" -Value 'y'
 Start-Process -FilePath $exeToRun -ArgumentList $statement -RedirectStandardInput "$env:TEMP\vimuninstallinput" -Wait -WindowStyle Hidden
 Remove-Item "$env:TEMP\vimuninstallinput"
-Remove-Item "$env:windir\vimtutor.bat"
+
```

Comment:
I think we still would want to have this removement, but hidden behind a call to `Test-Path` in the case of an old version of the package was installed in the past.

Reply:
I think `vimtutor.bat` is removed by `uninstall.exe` at line 9.

I tested with chocolatey-test-environment both scenarios:
1. Upgrade and uninstall
  After an upgrade from an old version to the new version with this PR, `vimtutor.bat` was replaced with the correct one. I uninstalled the package and it was deleted.
2. Clean install and uninstall
  I installed the new version in clean environment. When I uninstall it, the batch file is deleted as well.

Reply:
So, I think we don't need this Remove-Item anymore. Am I right? Or, missing some scenarios?
If you agree with me, please resolve this.

Reply:
My comment was made based on what was already available, since the installer itself removes the executable you are right this call is no longer needed.